app/policies/division_policy.rb

```diff
@@ -18,4 +18,30 @@ def update?
   def destroy?
     division_admin(division: @record)
   end
+
+  def select?
+    show?
+  end
+
+  class Scope < Scope
```

Comment:
Okay.  As I was verifying I discovered that Mel had actually already implemented quite a bit of unit testing, including `show` verification which was already dependent on the scope logic.  I'll go ahead and add some additional tests specific to scope.